zone/zoning.cpp

```diff
@@ -446,27 +446,34 @@ void Client::ProcessMovePC(uint32 zoneID, uint32 instance_id, float x, float y,
 
 	switch(zm) {
 		case GateToBindPoint:
+			cheat_timer.Start(3500, false);
```

Comment:
I would like to see the 3500 a constant that we reference somewhere, preferably if we don't have a dedicated class managing this logic I would prefer to see it in `emu_constants.h`, something like `EQEmu::Warp::GracePeriodNormal`